spec.bs

```diff
@@ -2659,20 +2659,59 @@ algorithms to achieve the outcomes described in the above explanatory content.
   or <{fencedframe}>'s [=container policy=], when it is constructed.
 </div>
 
+<div algorithm>
+  Create a new algorithm, called <dfn export>Create a Permissions Policy for a fenced
+  navigable</dfn>.
+
+  Given null or an [=element=] (|container|), this algorithm returns a new Permissions Policy.
```

Comment:
Done. The "Create a Permissions Policy for a navigable from response" patch should also reference this permissions policy, rather than the Document/permissions policy that it's currently referencing.

One note is that the current "Create a Permissions Policy for a navigable" spec capitalizes "Permissions Policy", but that seems wrong.